specification/cognitiveservices/AnomalyDetector/tspconfig.yaml

```diff
@@ -0,0 +1,26 @@
+emit: [
+  "@azure-tools/typespec-autorest",
+  "@azure-tools/typespec-python",
+  "@azure-tools/typespec-csharp",
+]
+options:
+  "@azure-tools/typespec-autorest":
+    "output-file": "openapi.json"
+    "examples-directory": ./examples
+  "@azure-tools/typespec-python":
+    "basic-setup-py": true
+    "package-version": 3.0.0b6
```

Comment:
@catalinaperalta do you know if we actually need this version here?

Reply:
This PR is for typespec renaming. Ignore this now. and this is specifical for python sdk package version naming, and will contact python sdk person to refine this if needed.